src/DynamicRupture/Output/Builders/PickPointBuilder.hpp

```diff
@@ -107,8 +107,8 @@ class PickPointBuilder : public OutputBuilder {
                                                           meshVertices[element.vertices[3]].coords,
                                                           receiver.global.getAsEigenVector());
         receiver.isInside = true;
-
       } else {
+        logInfo() << "pickpoint fault output: receiver (" << receiverId << ") is not inside";
```

Comment:
Can you clarifiy:
is not inside what? The mesh? volume cells adjacent to the faults?

Reply:
Added the following:
```
        logInfo() << "pickpoint fault output: receiver (" << receiverId << ") is not inside"
                  << "of any element along the rupture surface";
````
Does it sound better?

Reply:
ok.